cirq-core/cirq/qis/channels_test.py

```diff
@@ -80,14 +96,71 @@ def test_kraus_to_choi(kraus_operators, expected_choi):
     ),
 )
 def test_operation_to_choi(channel):
-    """Verifies that cirq.choi correctly computes the Choi matrix."""
+    """Verifies that cirq.operation_to_choi correctly computes the Choi matrix."""
     n_qubits = cirq.num_qubits(channel)
     actual = cirq.operation_to_choi(channel)
     expected = expected_choi(channel)
     assert np.isclose(np.trace(actual), 2 ** n_qubits)
     assert np.all(actual == expected)
 
 
-def test_choi_on_completely_dephasing_channel():
-    """Checks that cirq.choi returns the right matrix for the completely dephasing channel."""
+def test_choi_for_completely_dephasing_channel():
+    """Checks cirq.operation_to_choi on the completely dephasing channel."""
     assert np.all(cirq.operation_to_choi(cirq.phase_damp(1)) == np.diag([1, 0, 0, 1]))
+
+
+@pytest.mark.parametrize(
+    'kraus_operators, expected_channel_matrix',
+    (
+        ([np.eye(2)], np.eye(4)),
+        (
+            [
+                np.eye(2) / 2,
+                np.array([[0, 1], [1, 0]]) / 2,
+                np.array([[0, -1j], [1j, 0]]) / 2,
+                np.diag([1, -1]) / 2,
+            ],
+            np.array([[1, 0, 0, 1], [0, 0, 0, 0], [0, 0, 0, 0], [1, 0, 0, 1]]) / 2,
+        ),
+        (
+            [
+                np.array([[1, 0, 0], [0, 0, 1]]) / np.sqrt(2),
+                np.array([[1, 0, 0], [0, 0, -1]]) / np.sqrt(2),
+            ],
+            np.array(
+                [
+                    [1, 0, 0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0, 0, 0],
+                    [0, 0, 0, 0, 0, 0, 0, 0, 1],
+                ]
+            ),
+        ),
+    ),
+)
+def test_kraus_to_channel_matrix(kraus_operators, expected_channel_matrix):
```

Comment:
To avoid confusion, `expected_channel_matrix` should not be reused here since it's already the name of a function in this file.

Reply:
Renamed functions to use verbs in the name.